Gemfile

```diff
@@ -1,17 +1,18 @@
 # frozen_string_literal: true
+
 source "https://rubygems.org"
-ruby '2.5.3'
+ruby "2.5.3"
```

Comment:
Do we agree with this? 😬 I mean, I absolutely don't want to start a single quote versus double quote ~ debate ~. Most Ruby projects within dxw seem to use single quotes though, and given that was our natural instinct for this project as well, maybe we could configure `prettier` to treat Ruby code differently from SCSS (I don't know what the SCSS world conventions are.)

Reply:
This came from @pezholio's blog post at https://www.dxw.com/2020/04/the-factors-of-code-review/, which in turn (as far as I can see) comes from https://github.com/dxw/rails-template/blob/master/doc/architecture/decisions/0003-use-standard-rb.md.

Personally I'm more used to using single quotes unless I need variable interpolation, but this was just a straight adherence to the style as given by the linting tool.

Reply:
Alas! I haven't started a client project from scratch, so I've never had to refer to the decisions. 🙈 

> Double quotes for string literals - because pre-committing to whether you'll need interpolation in a string slows people down
1.9 hash syntax - When all the keys in a hash literal are symbols, Standard enforces Ruby 1.9's {hash: syntax}

But pre-committing to whether all my keys will always be symbols is slowing me down! :trollface: